dev/com.ibm.ws.ejbcontainer.remote.portable/bnd.bnd

```diff
@@ -11,6 +11,9 @@
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
 
+javac.source: 1.6
```

Comment:
this should be unset

Reply:
They should be 1.6 for ejb portable bundles.  I talked it over with @tkburroughs.  The bundle can be used with other products that could be using Java 1.6 still.

Reply:
Yes, the files in ejb portable are packed for use by the server, but also in a JAR that may be used by clients, such as a different vendor application servers.  It provides EJB remote interoperability between server products. It may also be used by older versions of Liberty or WebSphere traditional to interoperate with the latest version.  So, generally the classes need to be targed for an older Java level to support this backward compatibility.

Reply:
Ok, does this change need to be ported to 17003 stabilization then?  Previously we were compiling this module at 1.7